app/assets/v2/css/leaderboard.css

```diff
@@ -128,7 +147,21 @@
   }
 
   #leaderboard .winner img {
-    min-height: 18em;
+    display: flex;
+  }
+  .restext {
+    font-size: 0.7rem !important;
+  }
+  .restext2 {
+    font-size: 1rem !important;
+  }
+  .restext3 {
+    font-size: 1.2rem !important;
+  }
+  .restext4 {
+    font-size: 0.9rem !important;
+    font-weight: bold;
+    color: #CECFD1;
   }
```

Comment:
Same as [here](https://github.com/gitcoinco/web/pull/3933/files#r265255842)

Reply:
is this css repeated? 
Can we avoid using !important ?

Reply:
Ok I saw there are media queries, but can we go mobile first?  
Style first the mobile size and then use  min-width to style the big screens